### PR TITLE
Improves alignment for progress details overlay

### LINKED ISSUE #834: UI improvement: Progress details dropdown should be centered relatively to its parent
**Describe the bug**
Dropdown is not horizontally centered relatively to its parent

**To Reproduce**
1. As usual user, navigate to [this group](https://dev.algorea.org/en/#/groups/by-id/6710944276987033666;path=5121055722873780306/details)
2. Observe
3. Click on first suggested activity "Blockly basic task"
4. Navigate to "Progress" tab > "Chapter view"
5. Click on first result
6. See the dropdown position

**Expected behavior**
Dropdown should be centered horizontally or where the click happened.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by Iloveall**
## Description

Fixes #834

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

...

## Test cases

- [ ] Case 1:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/ui-improvement-center-dropdown-in-progress-details/en/#/groups/by-id/672913018859223173;path=52767158366271444/details)
  3. And start watch group 
  4. Then I click on "test 2 (Pixal)" of suggested list
  5. Then click on Progress tab -> chapter view tab
  6. And click on progress cell
  7. Then I see centred overlay

### CODE REVIEW

File: src/app/modules/item/pages/group-progress-grid/group-progress-grid.component.ts [hunk 1]
```diff
@@ -140,16 +140,17 @@ export class GroupProgressGridComponent implements OnChanges, OnDestroy {
...
+      this.progressOverlay = {
+        accessPermissions: {
+          title: row.header,
+          groupId: row.id,
+          itemId: col.id,
+        },
+        target,
+        progress: userProgress,
+      };
+    });
```
Comment (ThomasJuster) [inline src/app/modules/item/pages/group-progress-grid/group-progress-grid.component.ts:153]: Why did you need to change this piece of code?

**Review by ThomasJuster — COMMENTED** [review]
:+1:

File: src/app/modules/item/pages/group-progress-grid/group-progress-grid.component.ts (hunk 1 above)
Comment (Iloveall) [reply src/app/modules/item/pages/group-progress-grid/group-progress-grid.component.ts:153]: Otherwise, overlay doesn't work

**Review by smadbe — APPROVED** [review]
